smithy-swift-codegen/src/test/kotlin/software/amazon/smithy/swift/codegen/requestandresponse/requestflow/AuthSchemeResolverGeneratorTests.kt

```diff
@@ -42,28 +42,28 @@ public struct DefaultExampleAuthSchemeResolver: ExampleAuthSchemeResolver {
         }
         switch serviceParams.operation {
```

Comment:
Below is just test expectations that are changed due to changing `var` to `let`.